### DISCUSSION TITLE
Network interruption when executing the kickstart file.

### DISCUSSION THREAD

**Discussion by icebearrrr** [ipxe/ipxe#759] [Q&A]
I have a boot script file based on iPXE:
```shell
kernel vmlinuz ksdevice=${mac} ip=my_ip::my_gw:my_mask:::none nameserver=my_dns load_ramdisk=1 sshd vnc ks=myks
initrd initrd.img
boot
```
and a ks.cfg file with network config:
```text
network --bootproto static --ip my_ip --gateway my_gw --netmask my_mask --nameserver my_dns --hostname my_hostname --device=bootif --onboot yes --noipv6
url --url "http://mirrors.my.mirror"
...
```
Then i restart the server to enter pxe boot, after a while the server downloads the ks.cfg file successfully but throw some exception when executing it:

```text
Couldn't not resolve host: mirrors.my.mirror: Unknown error.
```

But when i go to the anaconda shell, and try to download the mirror manually:
```shell
wget -r http://mirrors.my.mirror
```
Everything just goes well.

I restart the server again(pxe boot) and keep ping the ip of it, found that when the ks.cfg was downloaded, there were a short network interruption(the ip is died), which may be the reason why the server can not download the mirror resource. Then after the same error thrown again(Couldn't resolve host error), the ip became alive.

This question has confused me a lot

**Comment by stappersg** [comment]
> This question has confused me a lot

Zoom in with a network sniffer. `tcpdump` or `wireshark` are fairly good available (and good :-).

**Comment by NiKiZe** [comment]
iPXE does not have any kickstart file. ;)
You might need to take this with whatever Linux distro you are using. (but you didnt say which)
Hopefully someone might however recognize the issue and be able to help you out.
